### PR TITLE
fix running tasks

### PR DESCRIPTION
**Pull request by jackschofield23**
Signed-off-by: Jack Schofield <jack.schofield@answerdigital.com>

### Description

Fixes # .

Fixes the running tasks to return currently running tasks.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [x] New tests added to cover the changes.
- [x] All tests passed locally.
- [ ] [Documentation comments](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments) included/updated.

### CODE REVIEW

File: src/WorkflowManager/Database/Repositories/TasksRepository.cs [hunk 1]
```diff
@@ -76,34 +76,34 @@ private static async Task EnsureIndex(IMongoCollection<WorkflowInstance> workflo
...
+        public async Task<(IList<TaskExecution>, long)> GetAllAsync(int? skip, int? limit)
         {
             try
             {
-                var builder = Builders<WorkflowInstance>.Filter;
-
-                var filter = builder.Eq("Tasks.Status", TaskExecutionStatus.Accepted);
-                filter &= builder.Ne("Tasks.Status", TaskExecutionStatus.Dispatched);
+                var filter = new FilterDefinitionBuilder<WorkflowInstance>().Where(x =>
+                x.Status != Status.Succeeded && x.Status != Status.Failed);
```
Comment (lillie-dae) [inline src/WorkflowManager/Database/Repositories/TasksRepository.cs:84]: are you missing canceled check here?

Comment (jackschofield23) [reply src/WorkflowManager/Database/Repositories/TasksRepository.cs:84]: This is for the workflowinstances we don't have cancelled for these. I'm checking cancelled for the tasks below however.

File: src/WorkflowManager/Database/Repositories/TasksRepository.cs [hunk 2]
```diff
@@ -76,34 +76,34 @@ private static async Task EnsureIndex(IMongoCollection<WorkflowInstance> workflo
...
 
-                var result = await _workflowInstanceCollection.Aggregate()
-                    .Match(filter)
-                    .Unwind<WorkflowInstance, WorkflowInstanceTasksUnwindResult>(wf => wf.Tasks)
```
Comment (lillie-dae) [inline src/WorkflowManager/Database/Repositories/TasksRepository.cs:90]: if your no longer using WorkflowInstanceTasksUnwindResult might be worth removing it?

**Review by lillie-dae — APPROVED** [review]

File: src/WorkflowManager/Database/Repositories/TasksRepository.cs (hunk 2 above)
Comment (jackschofield23) [reply src/WorkflowManager/Database/Repositories/TasksRepository.cs:90]: Good shout, removed

**Review by RemakingEden — APPROVED** [review]

**Comment by RemakingEden** [comment]
Closes #312

**Pull request merged**